cmd/daemon/apis/apis.go

```diff
@@ -0,0 +1,60 @@
+package apis
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/lunarway/release-manager/internal/log"
+	"go.opencensus.io/exporter/jaeger"
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/trace"
+)
+
+// All of the configuration necessary to run a fluxcloud API
+type APIConfig struct {
+	Server   *http.ServeMux
+	Client   *http.Client
+	Exporter Exporter
+	// Config   Config
+	Log *log.Logger
+}
+
+// Initialize API configuration
+func NewAPIConfig(e Exporter, logger *log.Logger) APIConfig {
+	return APIConfig{
+		Server: http.NewServeMux(),
+		Client: &http.Client{
+			Timeout:   120 * time.Second,
+			Transport: &ochttp.Transport{},
+		},
+		Exporter: e,
+		Log:      logger,
+	}
+}
+
+// Listen on addr
+func (a *APIConfig) Listen(addr string) error {
+	if os.Getenv("JAEGER_ENDPOINT") != "" {
+		exporter, err := jaeger.NewExporter(jaeger.Options{
+			CollectorEndpoint: os.Getenv("JAEGER_ENDPOINT"),
+			Process: jaeger.Process{
+				ServiceName: "fluxcloud",
+			},
+		})
+		if err != nil {
+			return err
+		}
+
+		trace.RegisterExporter(exporter)
+	}
```

Comment:
Stolen directly from fluxcloud. Maybe we should just remove it for now?

Reply:
I'm fine with removing it for now. I do however think it would be nice with the handlers being instrumented, now that we have jaeger instrumentation in much of the rest of the applications.

You left in the `ochttp.Handler{}` handler. Does it some how magically hook into our existing tracing setup through the OpenCensus packages?

Reply:
I'm really not sure about that :/